rcp/org.testeditor.rcp4.tcltestrun/src/org/testeditor/rcp4/tcltestrun/MavenExecutor.java

```diff
@@ -80,6 +80,7 @@ public int executeInNewJvm(String goal, String pathtoPom, String testParam) thro
 		logger.trace("Execute maven in new jvm with: {}", command);
 		ProcessBuilder processBuilder = new ProcessBuilder();
 		processBuilder.inheritIO();
+		processBuilder.directory(new File(pathtoPom));
```

Comment:
We usually use camel-case. So it would be nicer to name this variable `pathToPom`